conf/application.conf

```
@@ -327,7 +327,9 @@ backendAnalytics {
   uri = ${?DEV_WK_ANALYTICS_URI}
   key = ""
   sessionPause = 30 minutes
+  databaseEnabled = true
   verboseLoggingEnabled = false
+  wellKnownUris = ["https://webknossos.org|dingdong"] # flat list of uri-key pairs in the format: uri|key
```

```suggestion
  wellKnownUris = [] # flat list of uri-key pairs in the format: uri|key. Example: "https://webknossos.org|secretKey". These are allowed to send their analytics events to this server for storage.
```

(Should be empty by default, otherwise any instance with this config will accept data with this not-really-secret key)